envoy/stats/store.h

```diff
@@ -65,6 +66,21 @@ class Store : public Scope {
 
   virtual void forEachTextReadout(std::function<void(std::size_t)> f_size,
                                   std::function<void(Stats::TextReadout&)> f_stat) const PURE;
+
+  virtual void forEachHistogram(std::function<void(std::size_t)> f_size,
+                                std::function<void(Stats::ParentHistogram&)> f_stat) const PURE;
+  /**
+   * Iterate over all stats that need to be flushed for sink.
```

Comment:
doxygen params for these

Reply:
My older comments were actually not correct :-( Fixed.